README.md

```diff
@@ -1 +1,84 @@
 # hof-lookup-replacer
+
+A Node.js script for replacing db lookup tables as a scheduled job configurable to different HOF form services.
+
+The script retrieves a CSV formatted file from a given location, validates against given rules and then replaces a lookup table using the CSV rows. On success or failure the script will send appropriate notification messages.
+
+This service uses either [pg-promise](https://vitaly-t.github.io/pg-promise/index.html) or [knex.js](https://knexjs.org/) to handle db connection and query building. You can configure the client and model used using environment variables.
+
+## Script steps
+
+1. Scheduled job runs with service specific environment.
+2. Connect to database and retrieve the latest uploaded data's S3/file-vault URL from an SQL database.
```

Comment:
```suggestion
2. Connect to database and retrieve the latest uploaded S3 object/file-vault URL from an SQL database.
```
is that what you mean?

Reply:
Not exactly in that we won't automatically fail the job for a few invalid rows in the whole batch, have updated this line to try to make more sense anyway.

Reply:
Hi I still don't understand the sentence.  I would just remove the s3 bit and write, the latest filevault url if that's what you mean.  I thought grammatically it should be a SQL db but I've looked online and apparently an SQL is accepted too.  It doesn't make sense to me because S is not a vowel 🤷‍♂️

Reply:
I've updated this one as you suggest - I think to be proper it should be 'a SQL' so have updated...I can see why both are accepted though - 'an SQL' is a bit easier to say :)